ocp-config/prov-app/cm.yml

```diff
@@ -138,53 +139,78 @@ objects:
       ods.git-ref=${ODS_GIT_REF}
 
+      #############
+      # Job configurations can have the following properties:
+      #
+      # Mandatory:
+      # jenkinspipeline.[some-job-name].desc -> description of the Job
+      # jenkinspipeline.[some-job-name].repo -> repo name of the Job
+      #
+      # Optional:
+      # jenkinspipeline.[some-job-name].branch -> a branch differing from ods.git-ref
+      # jenkinspipeline.[some-job-name].jenkinsfile -> a path to the jenkinsfile differing from [some-job-name]/Jenkinsfile
+      #############
       # adminjobs configuration: create projects, delete-projects and delete-components
       jenkinspipeline.adminjobs.create-projects.desc=Create Project
-      jenkinspipeline.adminjobs.create-projects.url=opendevstack/ods-core.git#${ODS_GIT_REF}/create-projects/Jenkinsfile
+      jenkinspipeline.adminjobs.create-projects.repo=ods-core
+
       jenkinspipeline.adminjobs.delete-projects.desc=Delete openshift project
-      jenkinspipeline.adminjobs.delete-projects.url=opendevstack/ods-core.git#${ODS_GIT_REF}/delete-projects/Jenkinsfile
+      jenkinspipeline.adminjobs.delete-projects.repo=ods-core
+
       jenkinspipeline.adminjobs.delete-components.desc=Delete openshift components
-      jenkinspipeline.adminjobs.delete-components.url=opendevstack/ods-core.git#${ODS_GIT_REF}/delete-components/Jenkinsfile
+      jenkinspipeline.adminjobs.delete-components.repo=ods-core
 
       # Backend quickstarters
-      jenkinspipeline.quickstarter.beSpringBoot.desc=Backend - SpringBoot/Java
-      jenkinspipeline.quickstarter.beSpringBoot.url=opendevstack/ods-quickstarters.git#${ODS_GIT_REF}/be-java-springboot/Jenkinsfile
-      jenkinspipeline.quickstarter.dockerPlain.desc=Backend - Docker/Plain
-      jenkinspipeline.quickstarter.dockerPlain.url=opendevstack/ods-quickstarters.git#${ODS_GIT_REF}/docker-plain/Jenkinsfile
-      jenkinspipeline.quickstarter.beGolangPlain.desc=Backend - Golang
-      jenkinspipeline.quickstarter.beGolangPlain.url=opendevstack/ods-quickstarters.git#${ODS_GIT_REF}/be-golang-plain/Jenkinsfile
-      jenkinspipeline.quickstarter.bePythonFlask.desc=Backend - Python/Flask
-      jenkinspipeline.quickstarter.bePythonFlask.url=opendevstack/ods-quickstarters.git#${ODS_GIT_REF}/be-python-flask/Jenkinsfile
-      jenkinspipeline.quickstarter.beScalaAkka.desc=Backend - Scala/Akka
-      jenkinspipeline.quickstarter.beScalaAkka.url=opendevstack/ods-quickstarters.git#${ODS_GIT_REF}/be-scala-akka/Jenkinsfile
-      jenkinspipeline.quickstarter.beTypescriptExpress.desc=Backend - NodeJS/Express
-      jenkinspipeline.quickstarter.beTypescriptExpress.url=opendevstack/ods-quickstarters.git#${ODS_GIT_REF}/be-typescript-express/Jenkinsfile
+      jenkinspipeline.quickstarter.be-java-springboot.desc=Backend - SpringBoot/Java
+      jenkinspipeline.quickstarter.be-java-springboot.repo=ods-quickstarters
+
+      jenkinspipeline.quickstarter.docker-plain.desc=Backend - Docker/Plain
```

Comment:
Can we rename from `Backend - Docker/Plain` to `Docker/Plain`? This is neither backend or frontend ... and maybe then we should move to the "Other quickstarters" section.

Reply:
I do agree to @michaelsauter . But can you please change the description from from _Backend - Docker/Plain_ to to _Other Docker/Plain_ and please move the definition the section _other_ to line 211?